js/src/carousel.js

```diff
@@ -366,7 +366,7 @@ class Carousel {
     const activeIndex = this._getItemIndex(activeElement)
     const lastItemIndex = this._items.length - 1
     const isGoingToWrap = (isPrevDirection && activeIndex === 0) ||
-                            (isNextDirection && activeIndex === lastItemIndex)
```

Comment:
Can you revert any whitespace changes that unrelated to the PR so that the diff is as clean as possible?

Reply:
Done :+1: